devel/tig/Portfile

```diff
@@ -24,6 +24,7 @@ depends_lib         bin:git:git \
                     port:libiconv \
                     port:ncurses \
                     port:pcre2 \
+                    port:pkgconfig \
```

Comment:
pkgconfig is probably not required at runtime, so this needs to be in `depends_build` instead.

Reply:
fixed, thanks.